src/pages/forside/sections/4-personinfo/4-utbetalinger/endring/utenlandsk-bankkonto/felter/LandField.tsx

```diff
@@ -29,6 +29,7 @@ const LandField = () => {
       {...register("land", {
         required: formatMessage("validation.land.pakrevd"),
       })}
+      id="bankensland"
```

Comment:
Er denne nødvendig?

Reply:
Ja, for hvis ikke så blir labelId i NAVSelect "undefined_label". Denne settes på label som gjør at vi refererer med htmlFor.

Reply:
Ah, riktig, har faktisk gjort akkurat det samme i min egen branch når jeg tenker meg om 😅